### PR TITLE
MAINT-35382: Fix display of uploaded files cyrillic name from WebDav.

### PR DESCRIPTION
**Pull request by mdiha**
When a folder's name is in Cyrillic character it is transliterated so to display the correct folder name i made sure to display the folder's title instead since it is not transliterated.

### CODE REVIEW

File: exo.jcr.component.webdav/src/main/java/org/exoplatform/services/jcr/webdav/resource/CollectionResource.java [hunk 1]
```diff
@@ -217,7 +219,7 @@ public HierarchicalProperty getProperty(QName name) throws PathNotFoundException
    {
       if (name.equals(DISPLAYNAME))
       {
-         return new HierarchicalProperty(name, decodeValue(node.getName()) + (node.getIndex() > 1 ? "[" + node.getIndex() + "]" : ""));
+         return new HierarchicalProperty(name, decodeValue(StringUtils.equals(node.getPrimaryNodeType().getName(), NodeTypeUtil.NT_FOLDER) ? node.getProperty(NodeTypeUtil.EXO_TITLE).getString() : node.getName()) + (node.getIndex() > 1 ? "[" + node.getIndex() + "]" : ""));
```
Comment (ahamdi) [inline exo.jcr.component.webdav/src/main/java/org/exoplatform/services/jcr/webdav/resource/CollectionResource.java:222]: - to check if  a node is of a defined type, use node.isNodeType(Nodetype)
- Why should you use another proprty than names for NT_Folders ? isn't it the same for NT_UNSTRUCTRED or NT_FILE ?

File: exo.jcr.component.webdav/src/main/java/org/exoplatform/services/jcr/webdav/util/NodeTypeUtil.java [hunk 2]
```diff
@@ -36,6 +36,128 @@
 
 public class NodeTypeUtil
 {
+   final static public String NT_UNSTRUCTURED = "nt:unstructured";
```
Comment (ahamdi) [inline exo.jcr.component.webdav/src/main/java/org/exoplatform/services/jcr/webdav/util/NodeTypeUtil.java:39]: No need to all those properties as they belong to ECMS and there is no need to add them to JCR

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: exo.jcr.component.webdav/src/main/java/org/exoplatform/services/jcr/webdav/util/NodeTypeUtil.java (hunk 2 above)
Comment (mdiha) [reply exo.jcr.component.webdav/src/main/java/org/exoplatform/services/jcr/webdav/util/NodeTypeUtil.java:39]: Okay i will remove them.

File: exo.jcr.component.webdav/src/main/java/org/exoplatform/services/jcr/webdav/resource/CollectionResource.java (hunk 1 above)
Comment (mdiha) [reply exo.jcr.component.webdav/src/main/java/org/exoplatform/services/jcr/webdav/resource/CollectionResource.java:222]: For NT_Folders names in Cyrillic characters are transliterated so we need to display the title instead.

Comment (ahamdi) [reply exo.jcr.component.webdav/src/main/java/org/exoplatform/services/jcr/webdav/resource/CollectionResource.java]: Use node.isNodeType("nt:folder") instead

Comment (mdiha) [reply exo.jcr.component.webdav/src/main/java/org/exoplatform/services/jcr/webdav/resource/CollectionResource.java]: Done already

**Review by ahamdi — APPROVED** [review]
LGTM

**Pull request merged**